Cargo.lock

```diff
@@ -2205,7 +2205,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "0c2a198fb6b0eada2a8df47933734e6d35d350665a33a3593d7164fa52c75c19"
 dependencies = [
  "cfg-if",
- "windows-targets 0.52.4",
+ "windows-targets 0.48.5",
```

Comment:
Is this part of the change intentional?

Reply:
I was also surprised by this change! Working from a clean checkout I only ran `cargo update -p sysinfo`. Nonetheless it appears I can revert this specific change and limit it to `sysinfo`.